mobile/apps/artwork/components/bid/query.coffee

```diff
@@ -8,6 +8,12 @@ module.exports = """
     is_closed
     is_auction
     is_auction_promo
+
```

Comment:
Extra spaces are a little odd here, maybe we should just alphabetize everything above the `sale_artwork` bit?